internal/cmd/auth.go

```diff
@@ -179,6 +179,16 @@ func auth(cmd *cobra.Command, args []string, path string) error {
 			return fmt.Errorf("error persisting username on local config: %w", err)
 		}
 		fmt.Printf("✔  Success! Logged in as %s\n", username)
+
+		firstTime := settings.RegisterUse("auth_login")
```

Comment:
I'm going to change it because `RegisterUse` is not being counted if it is `--headless`.

Reply:
I think it's fine to only count calls that don't have --headless.